### ISSUE TITLE
blog and community links take you to 404 page

### ISSUE THREAD

**Issue by csantanapr** [knative/docs#3756]
## Expected Behavior
On the home page of mkdocs https://dev-knative.netlify.app/development/
Click into the tabs/links blog and the community should take you to the correct path URL
![image](https://user-images.githubusercontent.com/1094878/121273920-52d15700-c897-11eb-97c1-d2f1512b0915.png)

It should take you to
1. https://dev-knative.netlify.app//blog/
2. https://dev-knative.netlify.app/community/

## Actual Behavior
Problem is that it takes you to these instead sending user to a 404 error page
1. https://dev-knative.netlify.app/development/blog/
2. https://dev-knative.netlify.app/development/community/
![image](https://user-images.githubusercontent.com/1094878/121274026-9035e480-c897-11eb-9ce7-b48fb997686c.png)

## Additional Info
**Additional context**
This was working until recently. 

The mkdocs.yaml instruct the links to be created with the relative path up one parent in the current URL location
https://github.com/knative/docs/blob/cc72b2494249078a0acd9241ebe9e175d39df461/mkdocs.yml#L236-L237

If the HTML is render with parent location `../blog/` it works
```
      <a href="../blog/" class="md-nav__link">Read the Blog ➠</a>
```
if the HTML is not render with relative `blog/` it doesn't work
```
      <a href="blog/" class="md-nav__link">Read the Blog ➠</a>
```

**Comment by csantanapr** [comment]
/kind mkdocs
I did a diff on the pip dependencies comparing a working run vs. not working
![image](https://user-images.githubusercontent.com/1094878/121274334-4b5e7d80-c898-11eb-882f-a8c181980843.png)

**Comment by csantanapr** [comment]
cc @julz

**Comment by omerbensaadon** [comment]
Fixed in #3758?
